components/clamav/clamav-clamdtop.p5m

```diff
@@ -0,0 +1,53 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2016 Jim Klimov. All rights reserved.
+# Partially inspired by SFE manifest
+#   http://sfe.opencsw.org/localhostoi151a8/manifest/0/antivirus%2Fclamav@0.98,5.11-0.0.151.1.8%3A20131210T210011Z
+#
+
+# pull the manpages out of the component dir
+<transform file path=usr/share/man/(man3/.+$) -> set action.hash %<\1> >
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+set name=pkg.fmri \
+    value=pkg:/antivirus/$(COMPONENT_NAME)-clamdtop@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY), the clamdtop monitor"
+set name=pkg.description value="$(COMPONENT_SUMMARY), the clamdtop monitor"
+set name=info.classification value="org.opensolaris.category.2008:Applications/System Utilities"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license="GPL2"
+
+depend fmri=antivirus/clamav-core type=require
+### The above line should ensure common ClamAV dependencies du-jour
+### as well as "clamav" UID and GID
+### The clamdtop client requires a curses library.
+
+file path=usr/share/man/man1/clamdtop.1
+
+### Reserved for the case we'd have a 64-bit ncurses (pdcurses) later
```

Comment:
Seems that the Makefile should've defined `LDFLAGS.$(BITS)` to `-L` and `-R` proper libdir variants. Verifying locally...